langstroth/src/main/java/beekeeper/exploring/Exploring.java

```diff
@@ -0,0 +1,16 @@
+package beekeeper.exploring;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+/**
+ * @author willstuckey
+ * @date 3/3/17
+ * <p></p>
+ */
+@SpringBootApplication
+public class Exploring {
+    public static void main(String[] args) {
```

Comment:
Are you sure we need this? We have another main method in `src/main/java/beekeeper/langstroth/Langsthroth.java`, and gradle is yelling at me for trying to have multiple main methods.